fairscale/nn/misc/flatten_params_wrapper.py

```diff
@@ -122,15 +122,15 @@ def _flatten_params(self, flat_param: Optional[nn.Parameter] = None) -> None:
         # register the views as plain attributes
         self._unflatten_params_as_views()
 
-    def _get_param_views(self, flat_param: Tensor) -> Generator:
+    def get_param_views(self, flat_param: Tensor) -> Generator:
```

Comment:
since this is becoming an public method, can you please:

1. add docstring with proper doc
2. assert flat_param is valid before using it?